examples/ahfe.py

```diff
@@ -4,142 +4,112 @@
 
 from rdkit import Chem
 from rdkit.Chem import AllChem
-from fe import pdb_writer
-from fe import topology
-from fe.utils import get_romol_conf
-from md import builders
 
-from timemachine.lib import potentials, custom_ops
-from timemachine.lib import LangevinIntegrator
-
-import functools
-import jax
+import timemachine
 
 from ff import Forcefield
+from ff.handlers import openmm_deserializer
+from ff.handlers.deserialize import deserialize_handlers
 
-from ff.handlers import nonbonded, bonded, openmm_deserializer
+from fe import topology
+from fe.utils import get_romol_conf
 
-from ff.handlers.deserialize import deserialize_handlers
+from md import builders
+from md.moves import NPTMove
+from md.states import CoordsVelBox
 
-# 1. build water box
-# 2. build ligand
-# 3. convert to recipe
+from pathlib import Path
 
-# construct an RDKit molecule of aspirin
-# note: not using OpenFF Molecule because want to avoid the dependency (YTZ?)
-romol = Chem.AddHs(Chem.MolFromSmiles("CC(=O)OC1=CC=CC=C1C(=O)O"))
+from tqdm import tqdm
```

Comment:
We do not currently have tqdm as a requirement in the repo, should we add it?

Reply:
Ahh, I misremembered! (I remembered [pinning](https://github.com/proteneer/timemachine/pull/442#discussion_r645881764) the version in an earlier PR but it was unmerged.)

Can add tqdm as a dependency, remove the progress bar (just use prints), or use a different progress bar

Reply:
Removed in https://github.com/proteneer/timemachine/pull/580/commits/297fdec7509bd14e4175a211a14b6ca3be6964ec